sodacitylabs/boring-framework
# project: integrate jest

---
name: Pull Request
about: Ask for code to be merged into master
---

**Describe your Code**
Integrates Jest as the preferred testing framework

**Expected behavior**
New projects should use jest as the test framework and work OOB.

**Related Issues**
closes #115 

**How to test**
Steps to test and verify the code works as expected:
1. In the `links` folder, do a new project via `node ../src/cli/index.js new blog1`
2. Validate the folder structure has the new jest stuff
3. Go through the Getting Started guide but replace `bin/boring` with referencing the relative filepath to the CLI 
4. Make sure that you can run `npm test` even though it will fail intentionally
